scripts/release.mjs

```diff
@@ -0,0 +1,72 @@
+/**
+ * Release helper script.
+ * Required to be run on the release branch.
+ *
+ * It merges the release branch to the `develop` and `master` branches and pushes them, along with the created tags.
+ */
+import inquirer from 'inquirer';
+import {
+  displayErrorMessage,
+  displaySeparator,
+  spawnProcess,
+} from './utils/index.mjs';
+
+displaySeparator();
+
+(async() => {
+  // Initial verification prompt
+  const answers = await inquirer.prompt([
+    {
+      type: 'confirm',
+      name: 'continueRelease',
+      message: 'This script will take the current state of the release branch and merge, commit and push it to the' +
+        ' `master` branch (as well as the tags).\nContinue?',
+      default: true,
+    }
+  ]);
+
+  if (!answers.continueRelease) {
+    process.exit(0);
+  }
+
+  // Check if all the files are committed.
+  {
+    const processInfo = await spawnProcess('git status -s', true);
+    const output = processInfo.stdout.toString();
+
+    // If there are any uncommitted changes, kill the script.
+    if (output.length > 0) {
+      displayErrorMessage('There are uncommitted changes present. Exiting.');
+
+      process.exit(1);
+    }
+  }
+  {
+    // Check if we're on a release branch.
+    const processInfo = await spawnProcess('git rev-parse --abbrev-ref HEAD', true);
+    const branchName = processInfo.stdout.toString();
+
+    if (!branchName.startsWith('release/')) {
+      displayErrorMessage('You are not on a release branch.');
+      process.exit(1);
+    }
+
+    // Pull the recent changes from the `develop` and `master` branches, to prevent conflicts after merging.
```

Comment:
To make the release script more complete can we add the `changelog consume` command to the process? Something like:
```js
await spawnProcess('npm run changelog consume -- --date 2022-01-01 | cat');
await spawnProcess('git add CHANGELOG.md .changelogs/');
await spawnProcess('git commit -m "Generate CHANGELOG entries"');
```
(`cat` is here to disable TTY hence disable prompts)

Reply:
Good point, forgot about that. It should probably be added to the `freeze` script, rather then `release`.